### ISSUE TITLE
WARNING Unused array variable while used in a code 

### ISSUE THREAD

**Issue by Proxxx23** [friendsoftwig/twigcs#141]
Hello,

I have this piece of code:

```
{% set disclaimerOtherTypes = ["Standard", "Fördjupning", "Snackis", "Bildspel"] %}
        {% if articleType == "Tyck" %}
          {% if "Ledare" in categoriesNames %}
            {% if "KT-ledare" in tagsNames %}
              {% set disclaimerParameter = 'article-disclaimer-tyck-ledare-kt-ledare' %}
            {% elseif "Kuriren-ledare" in tagsNames %}
              {% set disclaimerParameter = 'article-disclaimer-tyck-ledare-kuriren-ledare' %}
            {% else %}
              {% set disclaimerParameter = 'article-disclaimer-tyck-ledare' %}
            {% endif %}
          {% else %}
            {% set disclaimerParameter = 'article-disclaimer-tyck' %}
          {% endif %}
        {% elseif articleType in disclaimerOtherTypes %}
          {% set disclaimerParameter = 'article-disclaimer-other' %}
        {% endif %}
```

But twigcs tells me:

_l.211 c.15 : WARNING Unused variable "disclaimerOtherTypes"._

As you can see, this variable is used:

`{% elseif articleType in disclaimerOtherTypes %}`

Isn't it a bug?

**Comment by Dodenis** [comment]
Fix in this PR #142

**Comment by OwlyCode** [comment]
Sorry I missed this issue, didn't see it in my emails. Thank you everyone for your help!
